logic/src/commonMain/kotlin/com/wire/kalium/logic/util/CurrentPlatform.kt

```diff
@@ -14,18 +14,15 @@
  *
  * You should have received a copy of the GNU General Public License
  * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
  */
 
-package com.wire.kalium.logic.feature.call.scenario
+package com.wire.kalium.logic.util
 
-import com.sun.jna.Pointer
-import com.wire.kalium.calling.callbacks.MissedCallHandler
-import com.wire.kalium.calling.types.Uint32_t
-import com.wire.kalium.logic.callingLogger
+expect class CurrentPlatform
 
-object OnMissedCall : MissedCallHandler {
-    override fun onMissedCall(conversationId: String, messageTime: Uint32_t, userId: String, isVideoCall: Boolean, arg: Pointer?) {
-        // NOTHING TO DO | This callback is not triggered by AVS
-        callingLogger.i("[onMissedCall] - conversationId: $conversationId | userId: $userId")
-    }
+enum class PlatformType(val value: String) {
+    ANDROID("Android"),
+    JVM("Jvm"),
+    DARWIN("Darwin")
```

Comment:
Strange enough this one looks like it removed `OnMissedCall` and renamed/added `CurrentPlatform` 🤔